### DISCUSSION TITLE
Heishamon PCB

### DISCUSSION THREAD

**Discussion by IgorYbema** [bisand/node-red-contrib-panasonic-comfort-cloud#4] [Show and tell]
Hi,
Do you now the Heishamon PCB? It allows to intergrate directly to the Aquarea heatpump, so without panasonic cloud.
See my store https://www.tindie.com/products/thehognl/heishamon-communication-pcb/ or the github repo https://github.com/Egyras/HeishaMon

Would you be interested in a free PCB so you might be able to develop a node red contrib just like your comfort cloud contrib?

regards, Igor

**Comment by bisand** [comment]
> 
> 
> Hi, Do you now the Heishamon PCB? It allows to intergrate directly to the Aquarea heatpump, so without panasonic cloud. See my store https://www.tindie.com/products/thehognl/heishamon-communication-pcb/ or the github repo https://github.com/Egyras/HeishaMon
> 
> Would you be interested in a free PCB so you might be able to develop a node red contrib just like your comfort cloud contrib?
> 
> regards, Igor

Hi Igor,

It sounds interesting, but at the moment I do not have too much time available to take on a project like this. On the other hand I am willing to help you get the Node-Red module up on its feet by creating the initial repository and some deployment scripts. I can also be a participant in this module, but will have to rely on someone else to drive the development forward.

Please let me know if that's ok with you.

Regards,
André

**Comment by IgorYbema** [comment]
I now notice I misjudged your nodered contrib. I thought it was for Panasonic aquarea heatpumps already but it is for airconditioners. That is a too big change for you as you might not have a heatpump yourself.

**Reply by bisand** [reply]
Ok, I only have a Panasonic CU-HZ25UKE-5 air-to-air heat pump (aka air conditioner). But let me know if you need any tips and tricks regarding development of Node-Red modules.

**Comment by reivax2** [comment]
Hi Bisand! Yes, what you're mentioning is definitely possible. I have it set up myself and it works great. Check this out: 
[url] https://payhip.com/GuiasAerotermiaYsolar [/url]
